src/utils/string.ts

```diff
@@ -3,13 +3,11 @@
  * "  luna    +test+1 " => "luna+test+1"
  */
 export function searchify(keyword: string) {
-    return keyword
-        .replace(/[^a-zA-Z0-9\s]/g, ' ')
```

Comment:
These replacing cannot be removed. The intention is to replace special characters with white spaces, then all white spaces will be replaced to "+". What we need is to fix this regex to specific the character set.